src/web/tools/GestureHandlerOrchestrator.ts

```diff
@@ -32,9 +32,14 @@ export default class GestureHandlerOrchestrator {
   }
 
   public removeHandlerFromOrchestrator(handler: IGestureHandler): void {
-    this.gestureHandlers.splice(this.gestureHandlers.indexOf(handler), 1);
-    this.awaitingHandlers.splice(this.awaitingHandlers.indexOf(handler), 1);
-    this.awaitingHandlersTags.delete(handler.getTag());
+    if (this.gestureHandlers.includes(handler)) {
+      this.gestureHandlers.splice(this.gestureHandlers.indexOf(handler), 1);
```

Comment:
You can save the index to a variable and check if it's `-1` so you don't look through the array twice (same below).

Reply:
Done in [5f00c22](https://github.com/software-mansion/react-native-gesture-handler/pull/2831/commits/5f00c2221ffab64ead24c8b841cc065566eccbc6).